polaris.shopify.com/src/pages/examples/text-display-large.tsx

```diff
@@ -0,0 +1,13 @@
+import { Text } from "@shopify/polaris";
+import React from "react";
+import { withPolarisExample } from "../../components/PolarisExamplePage";
+
+function TextExample() {
+  return (
+    <Text variant="displayLg" as="h1">
+      Good evening, Dominic.
```

Comment:
```suggestion
      Good evening, Someone awesome on this project.
```

Let's celebrate some of the awesome people working on the type component by putting their names in the examples.

Reply:
I like this idea!